### PR TITLE
[14.0][FIX] connector_search_engine: fix sync_state not changed

### PR DESCRIPTION
**Pull request by hparfr**
when active is switched in a _compute_method write() is not called

### CODE REVIEW

File: connector_search_engine/models/se_binding.py [hunk 1]
```diff
@@ -77,16 +79,12 @@ def create(self, vals):
...
-            super(SeBinding, not_new).write(new_vals)
-
-        res = super(SeBinding, self - not_new).write(vals)
-        return res
+    @api.depends("active")
+    def _compute_sync_state(self):
+        for rec in self:
+            if rec.sync_state != "new":
+                rec.sync_state = "to_update"
+                rec.date_modified = fields.Datetime.now()
```
Comment (sebastienbeau) [inline connector_search_engine/models/se_binding.py:87]: it seem that we miss to implement the date_modified behaviour (it's use nowhere)
Maybe the best is to also create a computed field for date_modified

```
@api.depends("sync_state")
def _compute_date_modified(self):
    for rec in self:
        rec.date_modified = fields.Datetime.now()
```

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review)

**Comment by sebastienbeau** [comment]
can you just take the test from here https://github.com/OCA/search-engine/pull/101/files

File: connector_search_engine/models/se_binding.py (hunk 1 above)
Comment (ivantodorovich) [inline connector_search_engine/models/se_binding.py:87]: if written in a computed method, `date_modified` should also be made a computed field too

alternatively you could've changed `write` to `_write`, that is called when computed fields are recomputed.

**Comment by mourad-ehm** [comment]
@hparfr could you reopen this PR ?

**Comment by hparfr** [comment]
I will let this pr as is because, in v16 many things have changed and this one will not be ported.

**Pull request closed without merging**